File: backend-nestjs/auth-service/src/organization/organization-user.entity.ts
import { Entity, Column, PrimaryColumn, CreateDateColumn } from 'typeorm';

@Entity('organization_users')
export class OrganizationUserEntity {
  @PrimaryColumn({ name: 'org_id', type: 'bigint' })
  orgId: number;

  @PrimaryColumn({ name: 'user_id', type: 'bigint' })
  userId: number;

  @Column({ name: 'tenant_id', type: 'bigint' })
  tenantId: number;

  @Column({ name: 'user_type', length: 20, default: 'staff' })
  userType: string;

  @CreateDateColumn({ name: 'created_at' })
  createdAt: Date;
}

File: backend-java/cdm-auth/src/main/java/com/cdm/auth/entity/OrganizationUserEntity.java
package com.cdm.auth.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@TableName("organization_users")
@Getter @Setter @NoArgsConstructor
public class OrganizationUserEntity {

    private Long orgId;
    private Long userId;
    private Long tenantId;
    private String userType = "staff";
}
